Wikipedia/Categories/MWKSection+DisplayHtml.m

```diff
@@ -53,7 +52,7 @@ - (NSString*)getHeaderTag {
                 self.sectionId,
                 self.anchor,
                 self.line,
-                pencilAnchor,
+                [self.anchor isEqualToString:@"TOC_Read_More"] ? @"" : [self getEditPencilAnchor],
```

Comment:
Need to pull @"TOC_Read_More" out to a const string - used in one other place on this diff...